resources/web/wwi/protoVisualizer/Node.js

```diff
@@ -344,12 +345,15 @@ export default class Node {
     model['baseType'] = rawBody.match(/\{\s*(?:%<[\s\S]*?(?:>%\s*))?(?:DEF\s+[^\s]+)?\s+([a-zA-Z0-9_\-+]+)\s*\{/)[1];
     model['parameters'] = {};
 
+    let hiddenField = false;
     while (!tokenizer.peekToken().isEof()) {
       const token = tokenizer.nextToken();
       let nextToken = tokenizer.peekToken();
-
       const restrictions = [];
-      if (token.isKeyword() && nextToken.isPunctuation()) {
+
```

Comment:
The `nextToken.isPunctuation()` check seems to have been removed, is it intended?